### PR TITLE
Fix/icons fonts

### LINKED ISSUE #1563: Cannot apply other icons except material-icons
**Vuestic-ui version:** 1.3.4
image

[https://codesandbox.io/s/9msnz?file=/src/App.vue:87-101](https://codesandbox.io/s/9msnz?file=/src/App.vue:87-101)

**Steps to reproduce**

**What is the expected behavior?**

**What is the current behavior?**

**Other information**

### LINKED ISSUE #1610: Update ionicons to the latest version
**Is your feature request related to a problem? Please describe.**
Related to the request [here](https://github.com/epicmaxco/vuestic-ui/pull/1591#discussion_r839767758).

**Describe the solution you'd like**
Update ionicons library version in the docs examples.

**Describe alternatives you've considered**
None.

**Additional context**

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
- Fixed codesandbox examples view:
example

- Removed not working example of entypo icons
- Updated icons to show differences between icons libraries: 
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

closes #1563 
close #1610

### CODE REVIEW

**Comment by asvae** [comment]
Looks completely correct in implementation. 💯

**Comment by rustem-nasyrov** [comment]
@asvae, please, can you review again?

File: packages/docs/src/config/icons-config/fonts.ts [hunk 1]
```diff
@@ -30,3 +30,49 @@ export const fonts: IconConfiguration[] = [
...
+  <script src="https://cdn.jsdelivr.net/npm/entypo@2.2.1/index.min.js"></script>
+`
+
+export const iconsConfig = `
+  import { createApp } from "vue"
+  import App from "./App.vue"
+  import { VuesticPlugin, createIconsConfig } from "vuestic-ui"
+  import 'vuestic-ui/dist/vuestic-ui.css'
+
+  const fonts = ${JSON.stringify(fonts)}
```
[the remark this answers is not in the document]
Comment (rustem-nasyrov) [reply packages/docs/src/config/icons-config/fonts.ts]: @RVitaly1978 `JSON.stringify` loses resolve functions and regex.

[the remark this answers is not in the document]
Comment (rustem-nasyrov) [reply packages/docs/src/config/icons-config/fonts.ts]: It's from old commit I guess. I decided to keep example configurations by their meaning.

File: packages/docs/src/config/icons-config/fonts.ts [hunk 2]
```diff
@@ -30,3 +30,49 @@ export const fonts: IconConfiguration[] = [
...
+  const app = createApp(App)
+    .use(VuesticPlugin, {
+      icons: createIconsConfig({ fonts }),
+    })
+
+  app.use(VuesticPlugin)
```
[the remark this answers is not in the document]
Comment (rustem-nasyrov) [reply packages/docs/src/config/icons-config/fonts.ts:92]: Which line 72?

File: packages/docs/src/config/icons-config/fonts.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (rustem-nasyrov) [reply packages/docs/src/config/icons-config/fonts.ts]: @RVitaly1978 I moved configuration to separate file.

**Review by RVitaly1978 — APPROVED** [review]

File: packages/docs/src/helpers/CodeSandboxIconsHelper.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
+
+export const iconsStyles = `
+  <link
```
Comment (asvae) [inline packages/docs/src/helpers/CodeSandboxIconsHelper.ts:3]: This part probably needs some polish.

We want to provide some comments on which part does what. i.e.

```html
<!-- Font Awesome 4 -->
```

- [ ] We want to keep only one version of font-awesome.
- [ ] For ionicons might make sense to update to latest version (current one is 5.*)

You might want to move that into another issue.

Comment (rustem-nasyrov) [reply packages/docs/src/helpers/CodeSandboxIconsHelper.ts:4]: @asvae upgraded the Ionicon version.

File: packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue [hunk 4]
```diff
@@ -2,39 +2,25 @@
...
+        <va-icon class="fas fa-book" />
+        <va-icon name="fas-book" />
       </p>
 
       <p>
         Ionic<br>
-        <va-icon class="icon ion-md-home" />
-        <va-icon name="ion-home" />
+        <va-icon name="ion-book-outline" />
+        <va-icon name="ion-book" />
```
Comment (asvae) [inline packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue:18]: The idea with 2 icons was:
* for first icon - we use attributes (class or ligature + class)
* for second item we use `name`

Here both use `name`.

First one should be something like `<va-icon name="ion-book" tag="ion-icon">`

File: packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue [hunk 5]
```diff
@@ -2,39 +2,25 @@
...
       </p>
   </div>
 </template>
-<style lang="scss">
-p {
-  i + i {
-    margin-left: 2rem;
-  }
+<style lang="scss" scoped>
```
Comment (asvae) [inline packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue:22]: Nice catch!

File: packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue (hunk 4 above)
Comment (rustem-nasyrov) [reply packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue:18]: @asvae updated example.

Comment (asvae) [reply packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue:18]: <va-icon name="ion-book-outline" /> ?

**Review by asvae — APPROVED** [review]
Awesome!

**Pull request merged**